src/po/fi.po

```diff
@@ -9519,7 +9358,7 @@ msgid "minimum number of screen lines for a fold to be closed"
 msgstr "vähimmäismäärä rivejä taitoksen sulkemiseen"
 
 msgid "template for comments; used to put the marker in"
-msgstr "malline kommenteille, käytetään merkintään"
```

Comment:
correct translation for template is malline: https://www.kielitoimistonsanakirja.fi/#/malline?searchMode=all https://en.wiktionary.org/wiki/malline, mallinne is just 2 person possessive form of malli (your models) nd doesn't make any sense in this context

Reply:
interesting, looks like whatever dictionary poedit is using for spell checking lacks malline by default. reverted